src/OrderingForm.js

```diff
@@ -0,0 +1,240 @@
+            <Vorname
+              type="text"
+              placeholder="Vorname"
+              name="nachname"
+              maxlength="20"
+            />
+          </Label>
+        </InputContainer>
+        <InputContainer>
+          <Label>
+            <HiddenLabelText>Nachname:</HiddenLabelText>
+            <Nachnahme
+              type="text"
+              placeholder="Nachname"
+              name="name"
+              maxlength="20"
+            />
+          </Label>
+        </InputContainer>
+
+        <InputContainer>
+          <Label>
+            <HiddenLabelText>Straße:</HiddenLabelText>
+            <Straße
+              maxlength="20"
+              type="text"
+              placeholder="Straße"
+              name="straße"
+            />
+          </Label>
+        </InputContainer>
+        <InputContainer>
+          <Label>
+            <HiddenLabelText>Postleitzahl:</HiddenLabelText>
+            <Postleitzahl
+              maxlength="20"
+              type="text"
+              placeholder="Postleitzahl"
+              name="postleitzahl"
+            />
+          </Label>
+        </InputContainer>
+        <InputContainer>
+          <Label>
+            <HiddenLabelText>Stadt:</HiddenLabelText>
+            <Stadt
+              maxlength="20"
+              type="text"
+              placeholder="Stadt"
+              name="stadt"
+            />
+          </Label>
+        </InputContainer>
+        <InputContainer>
+          <Label>
+            <HiddenLabelText>eMail:</HiddenLabelText>
+            <Mail
+              maxlength="20"
+              type="text"
+              placeholder="eMail Adresse"
+              name="email"
+            />
+          </Label>
+        </InputContainer>
+        <InputContainer>
+          <Label>
+            <HiddenLabelText>Anzahl:</HiddenLabelText>
+            <Amount maxlength="20" type="number" placeholder="0" name="menge" />
+          </Label>
+        </InputContainer>
+        <MessageContainer>
+          <Label>
+            <HiddenLabelText>Nachricht:</HiddenLabelText>
+            <Message
+              type="text"
+              cols="10"
+              rows="8"
+              placeholder="Nachricht"
+              name="nachricht"
+            />
+          </Label>
+        </MessageContainer>
+        <ButtonContainer>
+          <Button>Hol mich hier raus!</Button>
+        </ButtonContainer>
+      </Form>
+    </Wrapper>
+  )
+}
+
+const Wrapper = styled.section`
+  margin: 0 auto;
+  width: var(--main-width);
+  border: 1px solid white;
+  border-radius: 15px;
+  padding: 0px;
+  background-color: white;
+`
+
+const Form = styled.form`
+  display: grid;
+  justify-content: center;
+  align-items: center;
+  grid-template-areas:
+    'headline'
+    'vorname'
+    'nachname'
+    'straße'
+    'postleitzahl'
+    'stadt'
+    'email'
+    'menge'
+    'message'
+    'button';
+`
+
+const Label = styled.label``
```

Comment:
You are not adding any styles to this `Label` component, so you can just use the normal HTML `label` element.